README.md

```diff
@@ -331,6 +333,21 @@ Specific id for container. The container renders with a default id of `rdtsN` wh
 
 Use to ensure a own unique id when a simple counter is not sufficient, e.g in a partial server render (SSR)
 
+### label
+
+Type: `string`
+
+Adds `aria-labelledby` to search input when input starts with `#`, adds `aria-label` to search input when label has value (not containing '#')
+
+### enableKeyboardNavigation
+
+Type: `bool` (default: `false`)
+
+Adds navigation with `arrow` keys, `page down/up` / `home/end` and toggle of selection with `enter`. `Arrow/page up/down` also toggles open of dropdown if closed.
```

Comment:
Will it work uniformly across different OSes? In theory, it should but we know in practice things aren't always as simple. I can help test this on Windows, Mac but I'm wondering if we need to setup some integration tests (not sure if there are open source tools for doing this). Any way, food for thought.

Reply:
We use .key on the event. See https://www.w3.org/TR/uievents-key/#named-key-attribute-values
I hope this should be generic across OS and browsers.

There's one exception, to trigger toggle with space on <Toggle.
` if (e.key === 'Enter' || e.keyCode === 32)` where 32 is space. But this is actually only useful if keyboardnavigation is off since you can't tab there otherwise.

Reply:
Let me see if there are any tools out there that can help us set up some test suites. I don't think setting up test suites is a blocker for this PR though.

Reply:
Also, I can't seem to find anything - did we decide on keeping the flag `enableKeyboardNavigation`? I don't see any reason why we would need it - keyboard navigation should come naturally.

Reply:
I can remove it